app-modules/notification/src/Models/Concerns/HasSubscriptions.php

```diff
@@ -46,6 +46,6 @@ trait HasSubscriptions
      */
     public function subscriptions(): MorphMany
     {
-        return $this->morphMany(Subscription::class, 'subscribable');
+        return $this->morphMany(Subscription::class, 'subscribable')->whereHas('user');
```

Comment:
@danharrin I am curious your opinion on this. Technically I think this would solve anywhere in the app where we are trying to retrieve subscriptions where the related User got deleted.

But it feels strange to modify the relationship directly rather than deal with this where the issue is happening.

I am unsure.

Reply:
Yeah we don't have that right now. And if we did I guess we could chain on the relationship `->withTrashed()` or something like that to override it.

I'll add a ticket to the backlog to prune subscriptions where there is no User or Educatable